### PR TITLE
Travis CI: Python 3.9 release candidate 2

### PR DESCRIPTION
**Pull request by cclauss**

### CODE REVIEW

**Review by bittner — COMMENTED** [review]
Hi @cclauss,

that's nice from you to update the Travis pipeline. I'm afraid, though, it doesn't make sense to merge the change, for two reasons:

1. The Travis CI pipeline is legacy code. We moved to GitHub Actions recently, because they can cover all target OSes out-of-the-box. (We waited too long for Travis to make that work.) – The Travis code is still here, because some parts are not yet taken over to GitHub Actions, or they should inspire a follow-up implementation before getting rid of them entirely.
1. Python 3.9 is not a target yet. The application will be packaged independent from the Python version installed on any target OS. That's the default setting. If installed by PyPI this might be different, but we're not there yet (at Python 3.9 on local machines) anyway.

Bottom line: Please don't take this personally, but I have to close this PR. :worried: 

Thanks anyway for trying to be helpful! :100: :medal_military:

**Comment by cclauss** [comment]
Understood.  Makes sense the migration to Actions.  Python 3.9 is only two weeks from GA so I have been testing it out because it is more strict than 3.8 so it is good to ensure things run smoothly.

**Comment by bittner** [comment]
The real problem is, we don't have any actual tests. Those need to be added.
